### ISSUE TITLE
A11Y: Testing the whole site

### ISSUE THREAD

**Issue by tisto** [plone/volto#2543]
**Is your feature request related to a problem? Please describe.**

Run a11y (axe) on the whole website.

**Comment by BhavyaT-135** [comment]
Hey @tisto, I used the `axe dev tools` extension to test the deployed demo site. Mostly, the issues are focused on creating the texts and the links on the page more discernible by altering the color contrast. Once, the testing for a particular page is complete, the dev tools offer issues and solutions on how the UI can be improved. Here is a screenshot for the Home page after running the axe dev tools testing.

![image](https://user-images.githubusercontent.com/55613637/159153379-111ef904-f872-47df-bd82-0bfd5f8e8675.png)

Could you please elaborate on what needs to be done further?

**Comment by tisto** [comment]
@BhavyaT-135 this ticket is about running an external a11y check that crawls the entire website and creates a report on a11y violations. Running axe tools can be one of the options.

It is odd that axe reports violations on the front page. I was under the assumption that we have an a11y test for the Plone front page here:

https://github.com/plone/volto/blob/master/cypress/tests/core/a11y.js

If you are interested you could have a look at that particular test and figure out why its results differ from the results on the screenshot (though, I would recommend creating a new issue for that).
